.github/workflows/save_daily_metrics.yml

```diff
@@ -41,9 +41,7 @@ jobs:
         continue-on-error: true
         uses: "google-github-actions/auth@v2"
         with:
-          workload_identity_provider: "projects/345950277072/locations/global/workloadIdentityPools/gh-actions-pool/providers/gh-actions-provider"
-          service_account: "pudl-usage-metrics-etl@catalyst-cooperative-pudl.iam.gserviceaccount.com"
-          create_credentials_file: true
+          credentials_json: "${{ secrets.GCP_USAGE_METRICS_ARCHIVER_KEY }}"
```

Comment:
Hmm, sure, but what was the issue with WIF?

Reply:
This was done to give this archiver write access and to keep the ETL account as having read-only access to the archived files.

Reply:
Makes sense! Do you have a new service account now? Is it in Terraform / if not, do you want help putting it in Terraform?

Reply:
Let me move this question into the living terraform PR (https://github.com/catalyst-cooperative/pudl/pull/3841), as @bendnorman set up the account and would know better.